blocks.md

```diff
@@ -998,37 +997,36 @@ interface_name_include = []
 Key | Values | Required | Default
 ----|--------|----------|---------
 `primary_only` | Whether to show only the primary active connection or all active connections. | No | `false`
-`max_ssid_width` | Truncation length for SSID. | No | `21`
```

Comment:
Need to document that this has been removed. Perhaps better if we show what the new way is as well.